packages/@okta/vuepress-site/docs/reference/releases-at-okta/index.md

```diff
@@ -1,71 +1,72 @@
-| Contact with Product Team  | X                    |                     |                      |                      |            |
-| API  Changes               | Subject to change    | Subject to change   | Backwards compatible | Backwards compatible | N/A        |
-| Okta Support               |                      |                     | X                    | X                    |            |
-| Service-level agreements   |                      |                     | X                    | X                    | X          |
-| Announced in Release Notes |                      |                     | X                    | X                    | X          |
-| In preview orgs            | By invitation        | By invitation       | By request           | X                    | X          |
-| In production orgs         |                      |                     | By request           | X                    | X          |
-| Documentation              | Limited              | Limited             | X                    | X                    | X          |
+| Description                | Beta                 | EA                   | GA                   | Deprecated |
+| :------------              | :------------------: | :-----------------:  | :------------------: | :---------:|
+| Contact with Product Team  | &check;              | &cross;              | &cross;              | NA         |
+| API  Changes               | Subject to change    | Backwards compatible | Backwards compatible | N/A        |
+| Okta Support               | &cross;              | &check;              | &check;              | &check;    |
+| Service-level agreements   | &cross;              | &check;              | &check;              | &check;    |
+| Announced in Release Notes | &cross;              | &check;              | &check;              | &check;    |
+| In preview orgs            | By invitation or self-service, depending on the feature | By request or self-service, depending on the feature | &check;| &check;|
+| In production orgs         | &cross;              | By request or self-service, depending on the feature| &check;| &check; |
+| Documentation              | Limited              | &check;              | &check;              | NA         |
 
-## Beta
+## Beta Release
 
-Features in Beta are managed and supported by the Product Team and have been internally validated for a set of use cases.
-Minimal documentation is supplied for API-related Beta releases; API endpoint and configuration information is usually supplied.
+Features in Beta are managed and supported by the Product Team and have been internally validated for a set of use cases. Minimal documentation is supplied to customers for API-related Beta releases, and Beta documentation isn't available at developer.okta.com. API endpoint and configuration information is usually supplied.
```

Comment:
Would `developer.okta.com` be better as a link? Or how about something like `documentation isn't available on this website`?

Reply:
I'd prefer "Minimal documentation is directly provided to customers for API-related Beta releases, and is normally not publicly hosted."